indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -442,7 +442,8 @@
                 service-associations (get-service-associations context concept)
                 tool-associations (get-tool-associations context concept)
                 generic-associations (meta-db/get-generic-associations-for-concept context concept)
-                collection-associations (meta-db/get-associations-for-service context concept)
+                collection-associations (concat (meta-db/get-associations-for-service context concept)
+                                                (meta-db/get-associations-for-tool context concept))
```

Comment:
It's not necessary to add collection-associations. The way the code is written, it meant to use service-associations and tool-associations for the service and tool concepts, the same way as for the collection concept.  You just need to add the following. This way, it's consistent with the other concept types.
(defmethod get-service-associations :service
  [context concept]
  (meta-db/get-associations-for-service context concept)) 
(defmethod get-tool-associations :tool
  [context concept]
  (meta-db/get-associations-for-tool context concept))

Reply:
I have updated the code.